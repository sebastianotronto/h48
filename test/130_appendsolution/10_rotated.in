U2 F B L'
1
1
rotation UF
7
