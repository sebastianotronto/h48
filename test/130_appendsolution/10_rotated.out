D2 R L F'
Number of solutions: 1
Shortest solution length: 4
Used bytes: 10
